sps20/content/contents.lr

```diff
@@ -24,6 +24,18 @@ The conference will be held in English and is suited for all skill levels, from
 
 The summit is held on the **OST campus** in the beautiful town of **Rapperswil**. Find more information on the [venue](/venue) page.
 
+### Sponsors
+
+#### Gold
+
+#### Silver
+
+<a href="https://deplo.io"><img src="/sponsors/deploio.svg" style="height: 80px; width: 80px"></a>

```

Comment:
Our CSS for images is a bit wild in `sps20/assets/scss/style.scss` so I have to set explicit `width`.

Reply:
yes, thats true. i have a stall branch where i made some improves. i will try to fix things i  general when find time. but for this, i think its's fine!